courses/free-courses-ar.md

```diff
@@ -81,6 +82,7 @@
 * [Arabic C++ For kids & beginners (برمجة لصغار السن والمبتدئيين)](https://www.youtube.com/playlist?list=PLPt2dINI2MIbwnEoeHZnUHeUHjTd8x4F3) - Arabic Competitive Programming
 * [C++ Intensive - برمجة الحاسوب](https://www.youtube.com/playlist?list=PLPt2dINI2MIZPFq6HyUB1Uhxdh1UDnZMS) - Arabic Competitive Programming
 * [C++ Object-Oriented Design and Programming&rlm;](https://www.youtube.com/playlist?list=PLPt2dINI2MIbMba7tpx3qvmgOsDlpITwG) - Arabic Competitive Programming
+* [C++ Programming Basics&rlm;](https://www.youtube.com/playlist?list=PLv3VqjyehAoSSzkyHmWk89hPgcVwTNouG) - Ali Shahin
```

Comment:
Ali Shahin have a complete course in his website, he decided to move it in the youtube, so should I add `(:construction: *in process*` label?, it is already complete, he just moves each lesson every week or some days